BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/DocumentPages/DocumentPagesViewModelProtocol.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DocumentPagesViewModelProtocol.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+protocol DocumentPagesViewModelProtocol {
+    var rightBarButtonAction: (() -> Void)? { get }
+    var bottomInfoItems: [String] { get }
+
+    func processImages() -> [UIImage]
```

Comment:
[suggestion] What do you say about `imagesForDisplay` instead of `processImages`?